Confus/Confus/Game.cpp

```diff
@@ -54,8 +55,18 @@ namespace Confus
 			videoDriver->getTexture("Media/Textures/Heart.png"),
 			videoDriver->getTexture("Media/Textures/Blood.png"),
 			irr::core::vector2df(0.0f, 0.02f));
+        try
```

Comment:
But we would have problems with a lot of code if the path is invalid for other textures/models, I don't really know if this is the correct solution. Besides, the user shouldn't know if a crosshair was loaded or not.